charger/easee.go

```diff
@@ -413,9 +413,11 @@ func (c *Easee) Enable(enable bool) error {
 
 	// resume/stop charger
 	action := easee.ChargePause
+	expectedEnabledState := false
```

Comment:
```suggestion
	var expectedEnabledState
```

Das zero value wird nicht intialisiert.